packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -43,6 +43,49 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
    * URL that can be used to copy a meeting invite to the Users clipboard.
    */
   meetingInvitationURL?: string;
+  /**
+   * Call control options to change what buttons show on the meeting composite control bar.
+   */
+  meetingCallControlOptions?: MeetingCallControlOptions;
+}
+
+/**
+ * Meeting Call controls to show or hide buttons on the calling control bar.
+ *
+ * @beta
+ */
+export interface MeetingCallControlOptions
+  extends Pick<
+    CallControlOptions,
+    /**
+     * Flag that toggles the camera button in the MeetingCallControlBar component.
+     */
+    | 'cameraButton'
+    /**
+     * Flag that toggles the Microphone button in the MeetingCallControlBar component.
+     */
+    | 'microphoneButton'
+    /**
+     * Flag that toggles the Screen Share button in the MeetingCallControlBar component.
+     */
+    | 'screenShareButton'
+    /**
+     * Flag that toggles the Devices drop down and button in the MeetingCallControlBar component.
+     */
+    | 'devicesButton'
+    /**
+     * Selector that determines the display type to the CallControls component inside MeetingCallControlBar component, set to 'default' or 'compact'
+     */
+    | 'displayType'
+  > {
+  /**
+   * Flag for displaying the chat button inside the MeetingCallControlBar component.
```

Comment:
nit: "toggling" here is ambiguous - does it mean toggle as in check/uncheck or toggle whether it is shown or not

```suggestion
   * Show or Hide Chat Button in the meeting control bar.
   * @defaultValue true
```